src/content/configuration/dev-server.md

```diff
@@ -44,7 +44,7 @@ When the server is started, there will be a message prior to the list of resolve
 
 ```bash
 http://localhost:9000/
-webpack output is served from /build/
+the webpack output is served from /build/
```

Comment:
This doesn't look right, can you provide a screenshot of the output ?

Reply:
![image](https://user-images.githubusercontent.com/14764069/52529868-37ce9680-2d20-11e9-8d83-f3e9bd677a6f.png)

Reply:
@EugeneHlushko screenshot attached. See the highlighted text

Reply:
The output doesn't contain `the`, why did you post a screen shot of the `.md` file you edited?
I have asked a screenshot of the output you are getting after running latest version of the `webpack-dev-server` because i get different output after running it.